components/converter/Success.js

```diff
@@ -41,7 +48,6 @@ const Success = styled.div`
   }
   p {
     max-width: 410px;
-    margin: 0;
```

Comment:
These styling changes are pretty quick, but just wanted to include a bit of vertical padding against the stacked text.